appveyor.yml

```diff
@@ -50,3 +52,7 @@ environment:
   log_webhook_id: 382609729802862592
   SentryToken:
     secure: TSVcrUaf7lIi4YoOKrZhqrf6MvXh+XFUMO454dwB0q8ieNFOap49KhCYhw4x9sRflgQnDhppjzdbOrFiDnwQvID4rb3nXCxnJRzPLOUwOYlM4SfitMt2LnyGzyOdminq
+  DiscordClientId:
+    secure:
+  DiscordClientSecret:
+    secure: 
```

Comment:
be sure to modify the https://github.com/discord-csharp/CSDiscord/blob/master/before-deploy.ps1file to include these new environment variables.

You'll also need to expose the port that the web interface uses, map it to something other than 443/80

Reply:
We can change that with the `ASPNETCORE_URLS` environment variable - I'll add that, along with these to the configs. Also, you mean the `before-deploy.ps1` in *this* repo, right?